### DISCUSSION TITLE
Why I can't inherit one class from another?

### DISCUSSION THREAD

**Discussion by SergeyGuschin** [seleniumbase/SeleniumBase#1802] [General]
I I keep getting this error.

```
    from pages.home_page import HomePage
E   ModuleNotFoundError: No module named 'pages'
```

I have two py folders one folder named "pages" the other folder "tests" each folder has a py file **./pages/home_page.py**   and **./tests/test_home.py**

Here is my script for **home_page.py** 

```
from seleniumbase import BaseCase
class HomePage(BaseCase):
    def go_to_home(self):
        self.open("https://you.com/search?q=who+are+you&tbm=youchat&cfr=chat")
```

This is **test_home.py** 

```
from seleniumbase import BaseCase
from pages.home_page import HomePage

class MyTestClass(BaseCase):
    def test_page_open(self):
        home_page = HomePage(self)
        home_page.go_to_home_page()

```

**Comment by mdmintz** [comment]
Two issues. One, you named your method ``go_to_home()``, but then you called it with ``go_to_home_page()``. That's one issue. Second issue is that you need to correctly instantiate the class. Here's an example that uses the classic page object model: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/classic_obj_test.py

Here's how your code would look when using it:

```python
from seleniumbase import BaseCase

class HomePage:
    def go_to_home(self, sb):
        sb.open("https://you.com/search?q=who+are+you&tbm=youchat&cfr=chat")

class MyTestClass(BaseCase):
    def test_data_url_page(self):
        HomePage().go_to_home(self)
```

**Reply by SergeyGuschin** [reply]
Hi Michael. Thanks a lot for the quick response. I want to express my deep gratitude to you for the great work you've done and the contribution that you make to the community. I hope I am not mistaken if I thank you on behalf of all of us. Thank you very much.

**Reply by mdmintz** [reply]
Thank you, Sergey. It's great hearing about success stories with SeleniumBase. Let me know if you have any more questions.
